atom/browser/browser.h

```diff
@@ -216,6 +223,8 @@ class Browser : public WindowListObserver {
   std::string version_override_;
   std::string name_override_;
 
+  int current_badge_count_ = 0;
```

Comment:
Should this go behind a `#if defined(OS_LINUX)` check since those methods are behind that check?